src/signals/incident-management/containers/IncidentOverviewPage/constants.js

```diff
@@ -6,4 +6,18 @@ export const INCIDENT_SELECTED = 'sia/IncidentOverviewPage/INCIDENT_SELECTED';
 export const FILTER_INCIDENTS_CHANGED = 'sia/IncidentOverviewPage/FILTER_INCIDENTS_CHANGED';
 export const PAGE_INCIDENTS_CHANGED = 'sia/IncidentOverviewPage/PAGE_INCIDENTS_CHANGED';
 export const SORT_INCIDENTS_CHANGED = 'sia/IncidentOverviewPage/SORT_INCIDENTS_CHANGED';
-export const MAIN_CATEGORY_FILTER_SELECTION_CHANGED = 'sia/IncidentOverviewPage/MAIN_CATEGORY_FILTER_SELECTION_CHANGED';
+
+export const GET_FILTERS = 'sia/IncidentOverviewPage/GET_FILTERS';
+export const GET_FILTERS_SUCCESS = 'sia/IncidentOverviewPage/GET_FILTERS_SUCCESS';
+export const GET_FILTERS_FAILED = 'sia/IncidentOverviewPage/GET_FILTERS_FAILED';
```

Comment:
It might be time to get a Redux async triplet library to clean some of this up. It seems to cause most of the complexity in this PR at least.

Reply:
I'm quite pleased with: https://github.com/FredericHeem/redux-act-async

Reply:
we gaan het met front end meeting bespreken